raiden-dapp/src/services/raiden-service.ts

```diff
@@ -52,10 +52,19 @@ export default class RaidenService {
         this.store.commit('account', await this.getAccount());
         this.store.commit('balance', await this.getBalance());
 
-        this.subscription = raiden.channels$.subscribe(value => {
-          this.store.commit('updateChannels', value);
+        this.subscription = raiden.events$.subscribe(value => {
```

Comment:
You shouldn't need to unsubscribe manually, the `events$` stream completes upon shutdown, and therefore all subscriptions are dropped automatically. Also, I think it'd be more elegant to use a filter instead of a if, something like:
```
raiden.events$
  .pipe(filter(value => value.type === RaidenActionType.SHUTDOWN))
  .subscribe(() => this.store.commit('reset'));
```

Reply:
Ok it makes sense. I will go an remove them and then merge the PR.